editions/tw5.com/tiddlers/wikitext/HTML in WikiText.tid

```diff
@@ -14,7 +16,36 @@ This is my nice and simple block of text. HelloThere
 </article>
 ```
 
-[[Widgets share the same syntax as HTML tags|Widgets in WikiText]], and so the following information applies to them, too.
+!! Comments
+
+A html-comment as described above has been possible since ~TiddlyWiki Version 5.1.0.
+
+<<.from-version 5.2.0>> It will be possible to use "performant" ''html-comments together with pragmas''. Comments as described below can be cached by the TW core, so they are performant.
+
+```
+<!-- NEW: Comment that describes the macro -->
+\define test()
+some text <!-- inline comment -->
+\end
+
+<!-- This has always been possible -->
+<<test>>
+```
+
+If a macro use "text substitution" as described in [[Macro Definitions in WikiText]], you ''can use'' html-comments, ''but'' the will be ''slower'', since the core isn't allowed to cache them. Eg:
```

Comment:
I don't think it's worth confusing this explanation with the discussion about caching. Comments are cached in the same way as other constructions, and so I don't see anything special that we need to communicate here.